docs/howto/account-billing/pull-invoice-data.md

```diff
@@ -0,0 +1,65 @@
+# Pulling invoice data
```

Comment:
```suggestion
# Retrieving invoice data with the Cleura Cloud REST API
```

Reply:
Done!

Reply:
Nope, you can't use macros here. :) They work fine in the page title, but they break navigation.

Reply:
Argh, I forgot about that :/